shacl-node-expr/.prettierrc

```diff
@@ -0,0 +1,14 @@
+{
+  "useTabs": true,
```

Comment:
Let's not use tabs. They come differently for different people.

If they get mixed, then indentation can look quite irregular.

Does prettier work with reSpec? 

The usual problem I envcounter is the breaking lines inside links - legal but then the link e.g. a `data-cite` is unclear.